tensorboard/plugins/text/summary.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Text summaries and TensorFlow operations to create them.
+A text summary stores a single string value.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+from tensorboard.plugins.text import metadata
+
+
+def op(name,
+       data,
+       display_name=None,
+       description=None,
+       collections=None):
+  """Summarizes textual data.
```

Comment:
As with all other summary ops, let's be explicit that this is a TensorFlow op: "Create a text summary op." (For consistency, if nothing else.)

Reply:
Done.